docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,167 @@ sidebar_position: 1
+
+3. Check if operator has been installed successfully:
+
+```console
+$ k get csv -n operators
+NAME                              DISPLAY                    VERSION   REPLACES   PHASE
+service-binding-operator.v0.4.0   Service Binding Operator   0.4.0                Succeeded
+```
+
+4. To verify the installation, create a Service Binding:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+        name: binding-request
+spec:
+        services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+```
+
+The output should be:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"operators.coreos.com/v1alpha1","kind":"ServiceBinding","metadata":{"annotations":{},"name":"binding-request-backend","namespace":"default"},"spec":{"services":[{"group":"stable.example.com","kind":"Backend","name":"backend-demo","version":"v1"}]}}
+  creationTimestamp: "2021-05-17T07:09:48Z"
+  generation: 1
+  managedFields:
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .: {}
+          f:kubectl.kubernetes.io/last-applied-configuration: {}
+      f:spec:
+        .: {}
+        f:services: {}
+    manager: kubectl
+    operation: Update
+    time: "2021-05-17T07:09:48Z"
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:spec:
+        f:detectBindingResources: {}
+      f:status:
+        .: {}
+        f:conditions: {}
+        f:secret: {}
+    manager: service-binding-operator
+    operation: Update
+    time: "2021-05-17T07:09:48Z"
+  name: binding-request-backend
+  namespace: default
+  resourceVersion: "5795"
+  uid: 3c2114c9-311d-43a9-92e1-640179ab673a
+spec:
+  services:
+  - group: stable.example.com
+    kind: Backend
+    name: backend-demo
+    version: v1
+status:
+  conditions:
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    message: the server could not find the requested resource
+    reason: ServiceNotFound
+    status: "False"
+    type: CollectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "False"
+    type: InjectionReady
+  - lastHeartbeatTime: "2021-05-17T09:05:46Z"
+    lastTransitionTime: "2021-05-17T07:09:48Z"
+    status: "False"
+    type: Ready
+  secret: ""
+
+```
+
+## Install on Openshift
+
+1. Navigate to the `Operators` -> `OperatorHub` in the OpenShift console.
+
+2. Select `Developer Tools` category.
+
+3. Select the `Service Binding Operator`:
+
+![Service Binding Operator](https://github.com/redhat-developer/service-binding-operator/blob/master/assets/operator-hub-sbo-screenshot.png)
```

Comment:
The screenshot should be included in the sources of the documentation - we should not link to this repo.

Reply:
Removed outside link